src/utils/misc/getScrollLocation.js

```diff
@@ -0,0 +1,28 @@
+import ratio from "../basic/ratio";
+
+function getScrollRatio(endPageAdjustment = 0) {
+  // End page Adjustment means if you have some elements that you want to ignore
```

Comment:
`endPageAdjustement` - what's the unit of measure of this? the explanation talks about "having some elements", but i suspect it's just pixels? (also english in the explanation could be improved a bit) 

also, adjustment is normally either positive or negative. 

one possible alternative would be: 
- `shortenPageHeight`

maybe?